utils/feeLockHelper.ts

```diff
@@ -102,11 +102,47 @@ export async function calculateBuyPriceByMarket(
       JSON.stringify(
         await api.rpc.market.calculate_buy_price(
           poolId,
-          sellAssetId,
+          buyAssetId,
           sellAmount,
         ),
       ),
     ),
   );
   return value;
 }
+
+export async function calculateSellPriceWithImpact(
```

Comment:
Add rpc to the  begining of the function name, so its easier to find it later, and move it to Market static class.

Reply:
Done